tests/e2e/vm_disk_attachment_test.go

```diff
@@ -31,7 +31,10 @@ import (
 	kc "github.com/deckhouse/virtualization/tests/e2e/kubectl"
 )
 
-const APIVersion = "virtualization.deckhouse.io/v1alpha2"
+const (
+	APIVersion        = "virtualization.deckhouse.io/v1alpha2"
```

Comment:
Just to decrease hardcoded strings:

```suggestion
	APIVersion        = virtv2.SchemeGroupVersion.String()
```